python/ml4ir/applications/classification/tests/test_classification_model.py

```diff
@@ -14,25 +14,29 @@ def test_csv_loss_metrics(self):
         # Check if the loss, accuracy and top 5 accuracy on the test set is the same
         # Note that we don't check Precision which is not useful for this test model
         # Note that these numbers are different if you run it directly vs with docker-compose up
-        expected_loss = 1.6
+        expected_loss = 1.7
         tol = 0.2
-        self.assertTrue(np.isclose(self.metrics_dict["loss"], expected_loss, rtol=tol),
+        self.assertTrue(np.isclose(self.metrics_dict["loss"], expected_loss, atol=tol),
                         msg=f"Loss not in expected range."
                             f" Expected: {expected_loss} ± {tol}, Found: {self.metrics_dict['loss']}")
 
-        expected_acc = 0.203
-        tol = 0.01
-        self.assertTrue(np.isclose(self.metrics_dict["categorical_accuracy"], expected_acc, rtol=tol),
+        expected_acc = 0.2
+        tol = 0.05
+        self.assertTrue(np.isclose(self.metrics_dict["categorical_accuracy"], expected_acc, atol=tol),
```

Comment:
absolute points work better imho here as the numbers as small. Also, from a coders perspective is is easier to understand